app/src/pages/trade/claim.anc-ust-lp.tsx

```diff
@@ -117,18 +110,16 @@ function ClaimAncUstLpBase({ className }: ClaimAncUstLpProps) {
         {!!invalidTxFee && <MessageBox>{invalidTxFee}</MessageBox>}
 
         <TxFeeList className="receipt">
-          <TxFeeListItem label="Claiming">
-            {claiming ? formatANCWithPostfixUnits(demicrofy(claiming)) : 0} ANC
-            {' + '}
-            {userLPPendingToken
-              ? formatUToken(userLPPendingToken.pending)
-              : 0}{' '}
-            ASTRO
-          </TxFeeListItem>
-          <TxFeeListItem label="ANC After Tx">
-            {ancAfterTx ? formatANCWithPostfixUnits(demicrofy(ancAfterTx)) : 0}{' '}
-            ANC
-          </TxFeeListItem>
+          {hasAstroRewards && (
+            <TxFeeListItem label="ASTRO">
+              {formatUToken(astroRewards)} ASTRO
+            </TxFeeListItem>
+          )}
+          {hasAncRewards && (
+            <TxFeeListItem label="ANC">
+              {formatUToken(ancRewards as u<ANC<Big>>)} ANC
```

Comment:
why are we casting this one as u<ANC<Big>> but the astroRewards isnt? can we make this type the same as the astroRewards?

Reply:
✅ https://github.com/Anchor-Protocol/anchor-web-app/pull/317/commits/90fa938bf3e61a1159d328db8b0d869e167a0703